src/WingetCreateCLI/Properties/Resources.Designer.cs

```diff
@@ -1881,7 +1933,7 @@ public static string NestedInstallerParsing_HelpText
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to The title of the pull request submitted to GitHub.
+        ///   Looks up a localized string similar to The title of the pull request submitted to GitHub..

```

Comment:
>. [](http://example.com/codeflow?start=103&length=1)

nit: Is this extra dot auto generated by designer?

Reply:
Yes, this is auto generated because the string resource contains a period at the end.